modules/web-element.mjs

```diff
@@ -340,6 +340,20 @@ class FontFamily extends WebElementFunction{
         return s
     }
 }
+class checkColorGreen extends WebElementFunction{
```

Comment:
I don't understand the purpose of this class.

1. L352 extracts the color property based on a hard-coded path in the document; it should rather extract a property from the argument passed to the method (`element`)
2. Although its name is checkColor*Green*, what the method returns is the value of `color`

Reply:
this is a simple test for  the example that emile gave. I will make another class class or change the name.

Reply:
1. Do you mean this instruction : `var elem = doc.querySelector('.heading')`